tests/integration/api/test_sc_reports.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+import xml.etree.ElementTree as ET
+
+from tests.base import BaseTest
+
+from tenable_io.api.models import ScReport
+
+
+class TestScReportsApi(BaseTest):
+
+    @pytest.fixture(scope='class')
```

Comment:
I like this pattern, this actually would make sense from our PR yesterday. What do you think?